tests/e2e/constants/BaseTestConstants.ts

```diff
@@ -0,0 +1,65 @@
+/*********************************************************************
+ * Copyright (c) 2020-2023 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+export enum Platform {
+    OPENSHIFT = 'openshift',
+    KUBERNETES = 'kubernetes',
+}
+export const BaseTestConstants: any = {
```

Comment:
It's not clear what is the difference compare to UITestConstants class.
Would you, please, expand on this?

Reply:
This is constants are common to api and ui tests